app/src/main/java/co/develhope/chooseyouowncocktail_g2/RestClient.kt

```diff
@@ -0,0 +1,22 @@
+package co.develhope.chooseyouowncocktail_g2
+
+import com.google.gson.GsonBuilder
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RestClient {
+
+    private val gsonBuilder = GsonBuilder()
+        .setLenient()
+        .create()
+
+    private val retrofit =
+        Retrofit.Builder().baseUrl("https://www.thecocktaildb.com/api.php").addConverterFactory(
```

Comment:
la base url conviene sempre tenerla in una costante, immagina di avere base url diversi a seconda degli ambienti di sviluppo (DEV - STAGING - PROD)